webdriver_wharf/interactions.py

```diff
@@ -193,37 +193,27 @@ def destroy_all():
         destroy(c)
 
 
-def pull(image):
+def pull(image_name):
     global last_pulled_image_id
 
     # Add in some newlines so we can iterate over the concatenated json
-    output = client.pull(image).replace('}{', '}\r\n{')
-    # Check the docker output when running a command, explode if needed
-    for line in output.splitlines():
-        try:
-            out = json.loads(line)
-            if 'error' in out:
-                errmsg = out.get('errorDetail', {'message': out['error']})['message']
-                logger.error(errmsg)
-                # TODO: Explode here if we can't pull...
-                break
-            elif 'id' in out and 'status' in out:
-                logger.debug('{id}: {status}'.format(**out))
-        except:
-            pass
-
-    pulled_image_id = image_id(image)[:12]
+    image = client.images.pull(image_name)
```

Comment:
I wasn't aware that was possible. This needs to be documented, maybe in bright red blinking letters, since it's a pretty important backward-incompatible change.